### PR TITLE
DOMA-1882 replaced getAll for find in meter and meter reading acesses

### PR DESCRIPTION
**Pull request by nomerdvadcatpyat**
In the mobile application, requests for `meters` and `meter readings` take a very long time.

**Solution**
Use find, instead of getAll

### CODE REVIEW

File: apps/condo/domains/meter/utils/serverSchema/index.js [hunk 1]
```diff
@@ -33,11 +35,12 @@ const getAvailableResidentMeters = async (context, userId) => {
         const residentPropertyId = get(resident, ['property', 'id'])
```
Comment (SavelevMatthew) [inline apps/condo/domains/meter/utils/serverSchema/index.js:35]: find don't have nested structures...it stores only ids, like SQL table, so 
```suggestion
        const residentPropertyId = get(resident, ['property'])
```

File: apps/condo/domains/meter/utils/serverSchema/index.js [hunk 2]
```diff
@@ -33,11 +35,12 @@ const getAvailableResidentMeters = async (context, userId) => {
...
         const residentUnitName = get(resident, 'unitName')
 
-        const serviceConsumers = await ServiceConsumer.getAll(context, {
+        const serviceConsumers = await find('ServiceConsumer', {
             resident: { id: resident.id, deletedAt: null },
```
Comment (SavelevMatthew) [inline apps/condo/domains/meter/utils/serverSchema/index.js:39]: why consumers for each resident is unique query? Why not something like `{id_in: residentsIds}`

**Review by toplenboren — APPROVED** [review]

**Review by SavelevMatthew — APPROVED** [review]

File: apps/condo/domains/meter/schema/Meter.test.js [hunk 3]
```diff
@@ -813,16 +812,13 @@ describe('Meter', () => {
...
-                await Meter.getAll(client, { id: meter.id })
-            }
-
             // Test access after residents disconnected in worker
             // NOTE: This place is unstable. Because of parallel sub-requests inside Meter.getAll
             // we get different path in error payload, and get either organization or property error
             // in random order from request to request. So we skip testing path/value here.
             // Exception message and error type is enough.
-            await expectToThrowAccessDeniedError(testFunc, null)
+            const metersAfterResidentReconnects = await Meter.getAll(client, { id: meter.id })
```
Comment (akarjakin) [inline apps/condo/domains/meter/schema/Meter.test.js:820]: why not use `find` here too?

please add empty row after `const`blocks

File: apps/condo/domains/meter/schema/MeterReading.test.js [hunk 4]
```diff
@@ -1065,11 +1065,11 @@ describe('MeterReading', () => {
             await sleep(1500)
 
             // test access after residents reconnection worker task done
-            const testFunc = async () => {
-                await MeterReading.getAll(client, { id: meterReading.id })
-            }
+            const meterReadingsAfterResidentReconnect = await MeterReading.getAll(client, {
```
Comment (akarjakin) [inline apps/condo/domains/meter/schema/MeterReading.test.js:1068]: maybe use `find` ?

File: apps/condo/domains/meter/schema/Meter.test.js (hunk 3 above)
Comment (nomerdvadcatpyat) [reply apps/condo/domains/meter/schema/Meter.test.js:820]: in tests we usually use api from `testSchema` of this domain

**Pull request merged**